OIPA/common/download_file.py

```diff
@@ -0,0 +1,300 @@
+"""Downloads files from http or ftp locations.
```

Comment:
Why not use the package? https://pypi.python.org/pypi/fileDownloader.py/0.4.0 
handier to let others to keep it up-to-date :)